### ISSUE TITLE
Create Compost with a   Compost Heap bin

### ISSUE THREAD

**Issue by REPOSEECO** [StrangeLoopGames/EcoSuggestions#1090]
To drop organîc in the wold ok but why is there no  Compost Heap bin  
Later in the Game the Wastefilter create this but for the first Day a  Compost Heap bin woud be nice

**Comment by ThePiachu** [comment]
Yeah, a compost heap would be really useful for getting rid of things like Beet Greens, seeds, or whatever else you don't need. Just make it accept any crop, seed, etc. and spit out compost. Could be a simple T1 item that takes Fertilisers labour. This would let Fertilise specialists buy seeds that a lot of players tend to accumulate over time (like tree seeds).
